backend/plugin/advisor/mysqlwip/test/column_auto_increment_must_unsigned.yaml

```diff
@@ -1,46 +1,64 @@
-- statement: CREATE TABLE t(a INT UNSIGNED AUTO_INCREMENT)
+- statement: CREATE TABLE t(a INT UNSIGNED AUTO_INCREMENT);
   want:
     - status: SUCCESS
       code: 0
       title: OK
       content: ""
       line: 0
       details: ""
-- statement: CREATE TABLE t(a INT AUTO_INCREMENT)
+- statement: CREATE TABLE t(a INT AUTO_INCREMENT);
   want:
     - status: WARN
       code: 417
       title: column.auto-increment-must-unsigned
       content: Auto-increment column `t`.`a` is not UNSIGNED type
-      line: 2
+      line: 1
+      details: ""
+- statement: CREATE TABLE t(a INT SIGNED AUTO_INCREMENT);
+  want:
+    - status: WARN
+      code: 417
+      title: column.auto-increment-must-unsigned
+      content: Auto-increment column `t`.`a` is not UNSIGNED type
+      line: 1
+      details: ""
+- statement: CREATE TABLE t(a INT ZEROFILL AUTO_INCREMENT);
```

Comment:
Does we need to consider `ZEROFILL`?

> If you specify ZEROFILL for a numeric column, MySQL automatically adds the UNSIGNED attribute.

and it's deprecated in MySQL 8.0.17
> As of MySQL 8.0.17, the ZEROFILL attribute is deprecated for numeric data types, as is the display width attribute for integer data types.

Reply:
No need for now. It's fine.